tembo-pgmq-python/pyproject.toml

```diff
@@ -15,29 +15,12 @@ packages = [{include = "tembo_pgmq_python"}]
 [tool.poetry.dependencies]
 python = ">=3.8"
 psycopg = {extras = ["binary", "pool"], version = "^3"}
-pydantic = ">=1,<3"
 orjson = "^3"
+python-dotenv = "^1.0.1"
 
 [tool.poetry.group.dev.dependencies]
-mypy = "1.1.1"
-pandas = "^2.0.1"
-pytest = "^7.3.0"
-debugpy = "^1.6.7"
 black = "^23.3.0"
-numpy = "^1.24.3"
-matplotlib = "^3.7.1"
-jupyter = "^1.0.0"
-notebook = "^6.5.4"
-pandas-stubs = "^2.0.1.230501"
-sqlalchemy = "^2.0.13"
-psycopg2 = "^2.9.6"
-scipy = "^1"
-typer = {extras = ["all"], version = "^0.9.0"}
-types-psycopg2 = "^2.9.21.11"
-locust = "^2.16.1"
 ruff = "^0.3.1"
-pyyaml = "^6.0.1"
-seaborn = "^0.13.0"
 psycopg = {version = "^3.1.14", extras = ["binary", "pool"]}
```

Comment:
Was it intentional to remove all of the dev dependencies? the [benchmark scripts](https://github.com/tembo-io/pgmq/tree/main/tembo-pgmq-python/benches) depend on most of this stuff. The intention of them being in dev group is so that they are not required to run the main application.

Reply:
I added the minimum development dependencies to the bench group.

Reply:
Thank you! We will need to update `poetry.lock` as well since the dependencies are getting shuffled around. I think `poetry lock --no-update` should work, but I think it will give you an error that you can follow if you run `poetry install` with a new environment.